test/Makefile

```diff
@@ -133,7 +133,7 @@ runnable_test_results=$(addsuffix .out,$(addprefix $(RESULTS_DIR)/,$(runnable_te
 compilable_tests=$(wildcard compilable/*.d) $(wildcard compilable/*.sh)
 compilable_test_results=$(addsuffix .out,$(addprefix $(RESULTS_DIR)/,$(compilable_tests)))
 
-fail_compilation_tests=$(wildcard fail_compilation/*.d) $(wildcard fail_compilation/*.html)
+fail_compilation_tests=$(wildcard fail_compilation/*.d) $(wildcard fail_compilation/*.sh) $(wildcard fail_compilation/*.html)
```

Comment:
Looks like there were other untested shell scripts in fail_compilation :/

Reply:
Yep, one more, added 4 weeks ago (converted from .d): https://github.com/dlang/dmd/pull/9231/files#diff-c79d6f5eac3478fdb49375949df1ffa4

Reply:
Fixed it by trimming the trailing `\r` on Windows.